Visualizer/src/components/home/owner/OwnerForm.vue

```diff
@@ -30,11 +30,16 @@ const simulationConfig = useSimulationConfigStore();
 
 const owner = computed(() => simulationConfig.owners[props.ownerIndex]);
 
+const ownerProperties = computed(() => {
+  return simulationConfig.availableOwnersForAllocator.find((o) => o.name === owner.value.type);
+});
+
 /**
  * Whenever the selected ownerType changes, make sure the requirements for minimum
  * and maximum number of locations are met
  */
 const updateLocationsForOwner = () => {
+  owner.value.meta = ownerProperties.value.meta;
   if (owner.value.locations > owner.value.maxLocations) {
     owner.value.locations = owner.value.locations.slice(0, owner.value.maxLocations);
   }
```

Comment:
why is the `ownerProperties` variable needed?

Reply:
To overwrite the meta properties of a created owner with the meta properties of the available owner preset

Reply:
shouldn't then the maxLocations be overwritten as well?

Reply:
true true true